ruby/.rubocop_todo.yml

```diff
@@ -1,191 +1,26 @@
 # This configuration was generated by
 # `rubocop --auto-gen-config`
-# on 2023-10-31 15:43:37 UTC using RuboCop version 1.50.2.
+# on 2023-12-22 18:15:28 UTC using RuboCop version 1.56.3.
 # The point is for the user to remove these configuration records
 # one by one as the offenses are removed from the code base.
 # Note that changes in the inspected code, or installation of new
 # versions of RuboCop, may require this file to be generated again.
 
-# TODO: Oct '23 -> 10 files inspected, 63 offenses detected, 53 offenses autocorrectable
```

Comment:
if you leave these stylistic markers in it'll show we're slowly getting more conformant. It'll also warn us if we're getting less conformant